typings/index.d.ts

```diff
@@ -345,6 +347,288 @@ declare module 'discord.js' {
+			DARK_AQUA: 0x11806A
+			DARK_GREEN: 0x1F8B4C
+			DARK_BLUE: 0x206694
+			DARK_PURPLE: 0x71368A
+			DARK_VIVID_PINK: 0xAD1457
+			DARK_GOLD: 0xC27C0E
+			DARK_ORANGE: 0xA84300
+			DARK_RED: 0x992D22
+			DARK_GREY: 0x979C9F
+			DARKER_GREY: 0x7F8C8D
+			LIGHT_GREY: 0xBCC0C0
+			DARK_NAVY: 0x2C3E50
+			BLURPLE: 0x7289DA
+			GREYPLE: 0x99AAB5
+			DARK_BUT_NOT_BLACK: 0x2C2F33
+			NOT_QUITE_BLACK: 0x23272A
+		}
+		Status: {
+			READY: 0
+			CONNECTING: 1
+			RECONNECTING: 2
+			IDLE: 3
+			NEARLY: 4
+			DISCONNECTED: 5
+		}
+		OPCodes: {
+			DISPATCH: 0
+			HEARTBEAT: 1
+			IDENTIFY: 2
+			STATUS_UPDATE: 3
+			VOICE_STATE_UPDATE: 4
+			VOICE_GUILD_PING: 5
+			RESUME: 6
+			RECONNECT: 7
+			REQUEST_GUILD_MEMBERS: 8
+			INVALID_SESSION: 9
+			HELLO: 10
+			HEARTBEAT_ACK: 11
+		}
+		APIErrors: {
+			UNKNOWN_ACCOUNT: 10001
+			UNKNOWN_APPLICATION: 10002
+			UNKNOWN_CHANNEL: 10003
+			UNKNOWN_GUILD: 10004
+			UNKNOWN_INTEGRATION: 10005
+			UNKNOWN_INVITE: 10006
+			UNKNOWN_MEMBER: 10007
+			UNKNOWN_MESSAGE: 10008
+			UNKNOWN_OVERWRITE: 10009
+			UNKNOWN_PROVIDER: 10010
+			UNKNOWN_ROLE: 10011
+			UNKNOWN_TOKEN: 10012
+			UNKNOWN_USER: 10013
+			UNKNOWN_EMOJI: 10014
+			UNKNOWN_WEBHOOK: 10015
+			BOT_PROHIBITED_ENDPOINT: 20001
+			BOT_ONLY_ENDPOINT: 20002
+			MAXIMUM_GUILDS: 30001
+			MAXIMUM_FRIENDS: 30002
+			MAXIMUM_PINS: 30003
+			MAXIMUM_ROLES: 30005
+			MAXIMUM_REACTIONS: 30010
+			UNAUTHORIZED: 40001
+			MISSING_ACCESS: 50001
+			INVALID_ACCOUNT_TYPE: 50002
+			CANNOT_EXECUTE_ON_DM: 50003
+			EMBED_DISABLED: 50004
+			CANNOT_EDIT_MESSAGE_BY_OTHER: 50005
+			CANNOT_SEND_EMPTY_MESSAGE: 50006
+			CANNOT_MESSAGE_USER: 50007
+			CANNOT_SEND_MESSAGES_IN_VOICE_CHANNEL: 50008
+			CHANNEL_VERIFICATION_LEVEL_TOO_HIGH: 50009
+			OAUTH2_APPLICATION_BOT_ABSENT: 50010
+			MAXIMUM_OAUTH2_APPLICATIONS: 50011
+			INVALID_OAUTH_STATE: 50012
+			MISSING_PERMISSIONS: 50013
+			INVALID_AUTHENTICATION_TOKEN: 50014
+			NOTE_TOO_LONG: 50015
+			INVALID_BULK_DELETE_QUANTITY: 50016
+			CANNOT_PIN_MESSAGE_IN_OTHER_CHANNEL: 50019
+			CANNOT_EXECUTE_ON_SYSTEM_MESSAGE: 50021
+			BULK_DELETE_MESSAGE_TOO_OLD: 50034
+			INVITE_ACCEPTED_TO_GUILD_NOT_CONTAINING_BOT: 50036
+			REACTION_BLOCKED: 90001
+		}
+		VoiceStatus: {
+			CONNECTED: 0
+			CONNECTING: 1
+			AUTHENTICATING: 2
+			RECONNECTING: 3
+			DISCONNECTED: 4
+		}
+		VoiceOPCodes: {
+			IDENTIFY: 0
+			SELECT_PROTOCOL: 1
+			READY: 2
+			HEARTBEAT: 3
+			SESSION_DESCRIPTION: 4
+			SPEAKING: 5
+			HELLO: 8
+			CLIENT_CONNECT: 12
+			CLIENT_DISCONNECT: 13
+		}
+		ChannelTypes: {
+			TEXT: 0
+			DM: 1
+			VOICE: 2
+			GROUP: 3
+			CATEGORY: 4
+		}
+		ClientApplicationAssetTypes: {
+			SMALL: 1
+			BIG: 2
+		}
+		MessageTypes: [
+			'DEFAULT',
+			'RECIPIENT_ADD',
+			'RECIPIENT_REMOVE',
+			'CALL',
+			'CHANNEL_NAME_CHANGE',
+			'CHANNEL_ICON_CHANGE',
+			'PINS_ADD',
+			'GUILD_MEMBER_JOIN'
+		]
+		ActivityTypes: [
+			'PLAYING',
+			'STREAMING',
+			'LISTENING',
+			'WATCHING'
+		]
+		DefaultMessageNotifications: [
```

Comment:
It would be less accurate. The `DefaultMessageNotifications` has a set order which wouldn't be represented by this. It's the difference between
```js
type Constants.DefaultMessageNotifications[0] // 'ALL'
// becoming
type Constants.DefaultMessageNotifications[0] // 'ALL' | 'MENTIONS'
```
The change could be made, though. assuming the user only cared that it was an array of string that could be `.split` e.t.c, then it wouldn't be a problem. Do you me to change it to the simpler typings?